api/libproject/devstart.rb

```diff
@@ -984,23 +984,16 @@ def update_user_registered_status(cmd_name, args)
   common.run_inline %W{gcloud config set account #{op.opts.account}}
   header = "Authorization: Bearer #{token}"
   content_type = "Content-type: application/json"
-  payload = "{\"email\": \"#{op.opts.user}\"}"
+  payload = "{\"email\": \"#{op.opts.user}\", \"disabled\": \"#{op.opts.disabled}\"}"
   domain_name = get_auth_domain(op.opts.project)
-  if op.opts.action == "add"
-    common.run_inline %W{curl -H #{header} -H #{content_type}
+  common.run_inline %W{curl -X POST -H #{header} -H #{content_type}
       -d #{payload} https://api-dot-#{op.opts.project}.appspot.com/v1/auth-domain/#{domain_name}/users}
```

Comment:
Existing issue - but these assumptions don't hold up anymore. We should add an explicit API endpoint field to our configs in this file and use that.

Reply:
(the only outlier for now is that prod is: `api.workbench.researchallofus.org`)

Reply:
Yes, very true. Will do.

Reply:
Done for this API, but have not gone through to refactor all of the other api calls. Filing a ticket to make sure we cover this.